client/src/app/+login/login.component.scss

```diff
@@ -56,46 +31,24 @@ input[type=email] {
 }
 
 .alert {
-  margin: 0 auto 30px;
+  @include rfs(0 auto 2rem, margin);
```

Comment:
You don't use the `margin` peertube mixin?

Reply:
My mistake, corrected.